src/components/Books.js

```diff
@@ -0,0 +1,9 @@
+const Books = () => (
+  <div>
+    <h1> this is books</h1>
+    <button type="button">Remove a book</button>
+    <section className="book_list" />
+  </div>
```

Comment:
- [ ] As pointed out by the previous reviewer, please create another component that will be a single book component.

Reply:
Done, Thanks for your feadback